egs2/must_c_v2/st1/conf/tuning/train_st_ctc_md_conformer_asrinit_v3_noamp_batch50m_ctcsamp0.1.yaml

```diff
@@ -0,0 +1,121 @@
+batch_type: numel
+batch_bins: 25000000
+accum_grad: 2
+max_epoch: 20
+patience: none
+init: none
+best_model_criterion:
+-   - valid
+    - acc
+    - max
+keep_nbest_models: 10
+# use_amp: true       # automatic mixed precision (amp)
+num_workers: 4
+
+encoder: conformer
+encoder_conf:
+    output_size: 256
+    attention_heads: 4
+    linear_units: 2048
+    num_blocks: 12
+    dropout_rate: 0.1
+    positional_dropout_rate: 0.1
+    attention_dropout_rate: 0.1
+    input_layer: conv2d
+    normalize_before: true
+    macaron_style: true
+    rel_pos_type: latest
+    pos_enc_layer_type: rel_pos
+    selfattention_layer_type: rel_selfattn
+    activation_type: swish
+    use_cnn_module: true
+    cnn_module_kernel: 31
+
+hier_encoder: conformer
+hier_encoder_conf:
+    input_layer: null
+    num_blocks: 6
+    linear_units: 2048
+    dropout_rate: 0.1
+    positional_dropout_rate: 0.1
+    attention_dropout_rate: 0.1
+    normalize_before: true
+    macaron_style: true
+    rel_pos_type: latest
+    pos_enc_layer_type: rel_pos
+    selfattention_layer_type: rel_selfattn
+    activation_type: swish
+    use_cnn_module: true
+    cnn_module_kernel: 31
+
+decoder: transformer_md
+decoder_conf:
+    attention_heads: 4
+    linear_units: 2048
+    num_blocks: 6
+    dropout_rate: 0.1
+    positional_dropout_rate: 0.1
+    self_attention_dropout_rate: 0.2
+    src_attention_dropout_rate: 0.2
+
+extra_asr_decoder: transformer
+extra_asr_decoder_conf:
+    num_blocks: 6
+    linear_units: 2048
+    dropout_rate: 0.1
+    positional_dropout_rate: 0.1
+    self_attention_dropout_rate: 0.2
+    src_attention_dropout_rate: 0.2
+
+md_encoder: transformer
+md_encoder_conf:
+    output_size: 256
+    attention_heads: 4
+    linear_units: 2048
+    input_layer: null
+    num_blocks: 2
+    dropout_rate: 0.1
+
+model_conf:
+    asr_weight: 0.3
+    mt_weight: 0.0
+    mtlalpha: 0.3
+    st_mtlalpha: 0.3
+    lsm_weight: 0.1
+    length_normalized_loss: false
+
+optim: adam
+optim_conf:
+    lr: 0.002
+    weight_decay: 0.000001
+scheduler: warmuplr
+scheduler_conf:
+    warmup_steps: 25000
+
+frontend_conf:
+  n_fft: 400
+  hop_length: 160
+
+specaug: specaug
+specaug_conf:
+    apply_time_warp: true
+    time_warp_window: 5
+    time_warp_mode: bicubic
+    apply_freq_mask: true
+    freq_mask_width_range:
+    - 0
+    - 27
+    num_freq_mask: 2
+    apply_time_mask: true
+    time_mask_width_ratio_range:
+    - 0.
+    - 0.05
+    num_time_mask: 5
+
+ctc_sample_rate: 0.1
+
+ignore_init_mismatch: true
+init_param:
+    - "/exp/byan/working/espnet-ml/egs2/must_c_v2/asr1/exp/asr_train_asr_conformer_raw_bpe4000_sp/valid.acc.ave_10best.pth:encoder:encoder"
```

Comment:
Is it OK to add some absolute paths?
Is this an example and the users can change it depending on their environment?
If so, can you add some comments?